src/App.js

```diff
@@ -32,6 +33,7 @@ function App() {
               <Route path="/reward/coupons" element={<Coupons />} />
               <Route path="/user-home/join-challenge" element={<JoinChallange />} />
               <Route path="/user-home/join-challange-success" element={<JoinChallangeSuccess />} />
+              <Route path="/chat" element={<Chat />} />
```

Comment:
@ziyangcS2023 @YinzhuSU Please change the path to `/recycle/chat` to match the change in route names, thanks!